src/MembershipManager.php

```diff
@@ -399,6 +406,22 @@ protected function prepareConditionArray(array $value, array $default = NULL) {
     return array_unique($value);
   }
 
+  /**
+   * Stores the given list of membership IDs in the static cache backend.
+   *
+   * @param string $cid
+   *   The cache ID.
+   * @param array $membership_ids
+   *   The list of membership IDs to store in the static cache.
+   */
+  protected function cacheMembershipIds($cid, array $membership_ids) {
```

Comment:
Should we add this to MembershipManagerInterface?

Reply:
No I don't think this should be part of the public API, this is just an internal caching strategy. We should be able to switch to different caching without affecting any end user websites.